pkg/ibmcloudprovider/ibm_volume_provider.go

```diff
@@ -29,5 +28,5 @@ import (
 type CloudProviderInterface interface {
 	GetProviderSession(ctx context.Context, logger *zap.Logger) (provider.Session, error)
 	GetConfig() *config.Config
-	GetClusterInfo() *utils.ClusterInfo
```

Comment:
Did we confirm if this is being used by any other library. This is common library.

Reply:
Yes, Sameer checked this. This interface is used only by block csi driver. A similar interface is declared for file csi driver in it's own library.